daemon.yaml

```diff
@@ -1,15 +1,25 @@
 ---
 drivers:
   daemon:
+    featureMap:
+      global:
+        eventbus: redisqueue
     features:
       global:
+        - name: "eventbus"
+          required: true
         - name: "driver:redispubsub"
     drivers:
+      redisqueue:
+        name: redisqueue
+        type: builtin
+        handlerData:
```

Comment:
`handlerData` sounded rough. But, I kind of don't want to use `data` here, because I am using `data` in too many places.